contracts/fixed_price_rev1.1.scilla

```diff
@@ -462,7 +495,22 @@ procedure HandleFulfillOrder(
     };
     
     msgs = three_msgs msg_to_royalty_recipient msg_to_svc_fee_recipient msg_to_seller;
-    send msgs
+    send msgs;
+    
+    (* Paying out a commission if there is one *)
```

Comment:
Would it be better if this logic is placed together in the code block calculating the commission fee (line 459)? The brand commission fee and brand recipient should be checked together (both should be valid) before the fee is deducted and paid out.
If we do this we can call if there is no brand recipient/commission fee (or either one is not valid)
`msgs = three_msgs msg_to_royalty_recipient msg_to_svc_fee_recipient msg_to_seller;`
Otherwise, we call:
`msgs = four_msgs msg_to_royalty_recipient msg_to_svc_fee_recipient msg_to_seller msgt to brand_recipient;`